src/screens/SortationPutaway/PutawayEntryScreen.tsx

```diff
@@ -41,7 +41,7 @@ export default function PutawayEntryScreen() {
         getPutawayDetailsByContainerId(containerId, (response) => {
           if (response && !response.error) {
             const allTasks: SortationTask[] = response?.response?.data || [];
-            const filteredTasks = allTasks.filter((task) => task.status === 'IN_PROGRESS');
+            const filteredTasks = allTasks.filter((task) => task.status === 'IN_PROGRESS' || task.status === 'PENDING');
```

Comment:
I think we don't need the filtering anymore (unless we really don't want to see the STARTED tasks, if there are any eitherway), but I am merging as is